bin/run-test.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
 jshint . || exit 1
+for dir in test/hooks/fixtures/*/ ;
+do
+  cd ${dir} && npm install && cd -
```

Comment:
Add `echo "npm install in ${dir}"`.
For bonus marks, add `echo -en "travis_fold:start:npm install in ${dir}\\r"` along with an matching end.